charts/presto/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 name: presto
 description: The official Helm chart for Presto
 type: application
-version: 0.4.0
+version: 0.4.1
```

```suggestion
version: 0.4.0
```
The proposed change is not a bug fix and should be released in the next minor version of the chart. Therefore, please do not update the chart version, as that would trigger the release workflow.

---

done.